app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.scss

```diff
@@ -54,8 +54,8 @@
 
 .remove-issue-fields-button {
   position: absolute;
-  right: -4px;
-  top: 25px;
+  right: 5px;
+  top: 12px;
```

Comment:
seems like better align cross sign in the middle the `link to issue` field
please, discuss with task reporter
![image](https://user-images.githubusercontent.com/42434528/120621371-7bf08280-c466-11eb-85f8-94928ecc6f56.png)

Reply:
Fixed the position to the level of the upper border

Reply:
I thnk it was a bug that the crosses where placed not near the inputs.
The best choice here is (see the screenshot below)

![image](https://user-images.githubusercontent.com/31861805/120627405-4fd80000-c46c-11eb-90d8-d41fa30e3e51.png)

It is an initial placement for this crosses discussed with designer, so please move them according to the picture.